src/components/collection/DownloadButton.tsx

```diff
@@ -35,7 +37,7 @@ export const useDownloadAction = (itemId?: string) => {
 
   const startDownload = () => {
     if (itemId) {
-      exportZip({ id: itemId });
+      exportZip({ itemId, type: Triggers.ItemDownload });
```

Comment:
Is it necessary to add the type in the export zip? Can't this be infer from the backend?

Reply:
It can be actually